.rubocop.yml

```diff
@@ -1,6 +1,6 @@
 ---
 AllCops:
-  TargetRubyVersion: 2.3
+  TargetRubyVersion: 2.4
```

Comment:
Yesterday I pinned the version of rubocop rather than bump the minimum ruby version, I think if you rebase on master and just get rid of the commit fixing rubocop stuff this should be good to go

Reply:
We had the same thought at the same time! :-)